keps/sig-node/3008-qos-class-resources/README.md

```diff
@@ -472,6 +451,26 @@ usage of container-level QoS-class resources.
 
 Just using LimitRanges for specifying defaults could simplify the API.
 
+#### Implicit defaults
+
+If nothing is requested for a QoS-class resource, a pod/container still
+implicitly belongs to some class. By design there is no such thing as being in
+no QoS class. What this "unnamed" default class means or how it is handled is
+considered an implementation detail of the runtime. However, it would be
+potentially desirable for the user to be able to see what was classes the
+application was implicitly assigned to (even if nothing was explicitly
+requested). The first implementation phase contains no mechanism for this.
```

Comment:
So to be clear - the 1st phase is:
* pod says nothign about QoS
* pod gets scheduled
* pod gets assigned to some default
* user doesn't get to know what happened

?

I'm not sure I am comfortable with that.

Reply:
Yes, that's how it would work.

Concerns noted, I need to sleep over this and think about a solution to propose

Reply:
This should now be addressed e.g. in the "Implicit defaults" and "PodStatus" sections